Configurations/windows-makefile.tmpl

```diff
@@ -105,7 +105,7 @@ INSTALL_ENGINEPDBS={-
 INSTALL_FIPS={-
         join(" ", map { platform->dso($_) }
```

Comment:
@levitte INSTALL_ENGINES and other INSTALL_XXX variables quote their arguments ( `quotify1(platform->dso($_))`), while INSTALL_FIPS and INSTALL_MODULES don't do it. Is that on purpose, or an oversight?

Reply:
Looking at the generated Makefile I wonder whether it wouldn't be better to remove all those unnecessary quotes instead. They don't seem to contain user configurable parts which could contain spaces.

```
INSTALL_LIBS="libcrypto.lib" "libssl.lib"
INSTALL_SHLIBS="libcrypto-3-x64.dll" "libssl-3-x64.dll"
INSTALL_SHLIBPDBS="libcrypto-3-x64.pdb" "libssl-3-x64.pdb"
INSTALL_ENGINES="engines\\capi.dll" "engines\\padlock.dll"
INSTALL_ENGINEPDBS="engines\\capi.pdb" "engines\\padlock.pdb"
INSTALL_FIPS=providers\fips.dll
INSTALL_MODULES=providers\legacy.dll
INSTALL_MODULEPDBS="providers\\fips.pdb" "providers\\legacy.pdb"
INSTALL_PROGRAMS="apps\\openssl.exe"
INSTALL_PROGRAMPDBS="apps\\openssl.pdb"
BIN_SCRIPTS="tools\\c_rehash.pl"
MISC_SCRIPTS="apps\\CA.pl" "apps\\tsget.pl"
HTMLDOCS1=doc\html\man1\CA.pl.html doc\html\man1\openssl-asn1parse.html 
```

Reply:
It's an oversight...  INSTALL_FIPS and INSTALL_MODULES should have their arguments quote too.

> They don't seem to contain user configurable parts which could contain spaces

Believe you me, users may surprise us still...

Reply:
Done in d76ccbdf1b6d04591fff83b28a2233dd198c4e89.